index/secondary-index/src/test/scala/org/apache/carbondata/spark/testsuite/secondaryindex/TestSIWithPartition.scala

```diff
@@ -414,6 +414,7 @@ class TestSIWithPartition extends QueryTest with BeforeAndAfterAll {
     checkAnswer(extSegmentQuery, Seq(Row(2, "red", "def"), Row(5, "red", "abc")))
     assert(extSegmentQuery.queryExecution.executedPlan.isInstanceOf[BroadCastSIFilterPushJoin])
     sql("drop table if exists partition_table")
+    FileFactory.deleteAllCarbonFilesOfDir(FileFactory.getCarbonFile(sdkWritePath))
```

Comment:
can you add insert into existing external partition also

Reply:
Done